featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -238,7 +361,16 @@ def pd_avg_time_between(x):
 
 
 class Median(AggregationPrimitive):
-    """Finds the median value of any feature with well-ordered values."""
+    """Finds the median value of a feature.
+
+    Description:
+        Given a list of well-ordered values, return the median.
```

Comment:
what does well ordered mean?

Reply:
Removed this, and better short description.